### PR TITLE
feat: Re-export "AudioPlayer" component

### PR DESCRIPTION
**Pull request by Yakutoc**
Необходимо: 

1. Перенести компонент из **plasma-b2c** в **plasma-hope** и сделать **re-export** в 

   -  **plasma-b2c**
   -  **plasma-web**

2. Проверить дизайн в **plasma-web**;

:baby_chick: Download canary assets:

[colors--canary.369.4315448721.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.369.4315448721.xml)  
[color_metro_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.369.4315448721.swift)  
[color_metro_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.369.4315448721.kt)  
[color_metro_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.369.4315448721.ts)  
[color_sberHealth_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.369.4315448721.swift)  
[color_sberHealth_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.369.4315448721.kt)  
[color_sberHealth_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.369.4315448721.ts)  
[color_sbermarket_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.369.4315448721.swift)  
[color_sbermarket_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.369.4315448721.kt)  
[color_sbermarket_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.369.4315448721.ts)  
[color_sberprime_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.369.4315448721.swift)  
[color_sberprime_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.369.4315448721.kt)  
[color_sberprime_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.369.4315448721.ts)  
[color_selgros_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.369.4315448721.swift)  
[color_selgros_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.369.4315448721.kt)  
[color_selgros_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.369.4315448721.ts)  
[color_smbusiness_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.369.4315448721.swift)  
[color_smbusiness_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.369.4315448721.kt)  
[color_smbusiness_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.369.4315448721.ts)  
[PlasmaTokensColor--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.369.4315448721.swift)  
[shadow_sbermarket_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.369.4315448721.ts)  
[typo_mage_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.369.4315448721.swift)  
[typo_mage_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.369.4315448721.kt)  
[typo_mage_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.369.4315448721.ts)  
[typo_plasma_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.369.4315448721.swift)  
[typo_plasma_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.369.4315448721.kt)  
[typo_plasma_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.369.4315448721.ts)  
[typo_ruler_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.369.4315448721.swift)  
[typo_ruler_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.369.4315448721.kt)  
[typo_ruler_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.369.4315448721.ts)  
[typo_sage_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.369.4315448721.swift)  
[typo_sage_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.369.4315448721.kt)  
[typo_sage_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.369.4315448721.ts)  
[typo_sbermarket_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.369.4315448721.swift)  
[typo_sbermarket_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.369.4315448721.kt)  
[typo_sbermarket_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.369.4315448721.ts)  
[typo_soulmate_ios-swift--canary.369.4315448721.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.369.4315448721.swift)  
[typo_soulmate_kotlin--canary.369.4315448721.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.369.4315448721.kt)  
[typo_soulmate_react-native--canary.369.4315448721.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.369.4315448721.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.151.0-canary.369.4315448721.0
  npm install @salutejs/plasma-hope@0.10.0-canary.369.4315448721.0
  npm install @salutejs/plasma-web@1.176.0-canary.369.4315448721.0
  # or 
  yarn add @salutejs/plasma-b2c@1.151.0-canary.369.4315448721.0
  yarn add @salutejs/plasma-hope@0.10.0-canary.369.4315448721.0
  yarn add @salutejs/plasma-web@1.176.0-canary.369.4315448721.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-369/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-369/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-369/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-369/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-369/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-369/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-369/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-369/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-369/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-369/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-369/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-369/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-369/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-369/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-369/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Review by gibushnev — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-369/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-369/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-369/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-369/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-369/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-369/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-369/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Review by neretin-trike — APPROVED** [review]

File: packages/plasma-b2c/api/plasma-b2c.api.md [hunk 1]
```diff
@@ -330,21 +318,21 @@ contentRight?: undefined;
 children?: undefined;
 isLoading?: boolean | undefined;
 loader?: ReactNode;
-} & Partial<ButtonSizeProps<string | number>> & Partial<ButtonViewProps<"checked" | "primary" | "success" | "warning" | "critical" | "secondary" | "overlay" | "clear">> & RefAttributes<HTMLButtonElement>) | (ButtonProps_2<HTMLElement> & {
```
Comment (Yeti-or) [inline packages/plasma-b2c/api/plasma-b2c.api.md:333]: ну вот что за непостоянство

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-b2c/api/plasma-b2c.api.md (hunk 1 above)
Comment (Yakutoc) [reply packages/plasma-b2c/api/plasma-b2c.api.md:333]: @Yeti-or Я тоже пока не могу понять закономерность. Вроде ПР не трогает эти части системы... но тут удалить, то там добавит.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-369/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-369/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-369/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-369/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-369/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-369/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-369/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-369/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-369/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Pull request merged**